postings/models.py

```diff
@@ -61,6 +65,37 @@ def save_image(self, image):
         """
         pass
 
+    #Takes in a company name and bucket location for upload
+    #and uploads the file to S3, and returns the s3 url
+    def load_logo(self, company, bucket):
+        #Get company autocomplete and values
+        url = "https://autocomplete.clearbit.com/v1/companies/suggest?query=" + company
+        searchResult = urllib.request.urlopen(url)
+        data = searchResult.read()
+        encodedData = searchResult.info().get_content_charset('utf-8')
+        resultingData = json.loads(data.decode(encodedData))
+        
+        #Download Logo locally
+        for item in resultingData: 
+            clearbitURL = item["logo"]
+            imageFilename = company + ".png"
+            urllib.request.urlretrieve(clearbitURL, imageFilename)
+            break
+        
+        ACCESS_KEY = os.environ['ACCESS_KEY']
+        SECRET_KEY = os.environ['SECRET_KEY']
+        
+        #Upload to S3
+        s3 = boto3.client('s3', aws_access_key_id=ACCESS_KEY, aws_secret_access_key=SECRET_KEY)
+        s3.upload_file(imageFilename, bucket, imageFilename)
```

Comment:
use s3.upload_fileobj() instead of upload_file. This way you don't have to save the file to disk and then remove the file. Although upload_fileobj will require a buffer though. 

Starting point to look into: "https://stackoverflow.com/questions/45699905/csv-file-upload-from-buffer-to-s3"